fastlane/Fastfile

```diff
@@ -850,3 +875,62 @@ end
 def simulator_version
   '14.4'
 end
+
+def occupy_test_account
+  UI.message("Looking for a free test account...")
+  find_free_test_account
+  change_test_account_availability("T")
+end
+
+def deoccupy_test_account
+  return if $used_test_account_index == nil
+  UI.message("Freeing used test account")
+  change_test_account_availability("F")
+  $used_test_account_index = nil
+end
+
+# Returns the value of test accounts key, which is 
+# expected to look like a series of Fs and Ts.
+# Example:
+#   get_test_accounts_state
+#     => "TFFFF"
+def get_test_accounts_state
+  uri = URI.parse(UI_TESTS_ACCOUNTS_KEY)
+  response = Net::HTTP.get_response(uri)
+  accounts_state = response.body.gsub("\n","")
+  UI.message("State of test accounts: #{accounts_state}")
+  accounts_state
+end
+
+# Returns the index of first "F" (Free) occurence in the test
+# accounts keyvalue, or crashes fastlane if it was not found.
+# Example:
+#   get_free_test_account_index
```

Comment:
Nitpick. This documentation is out of sync with the name of the method.

I actually like this name better.

Reply:
Honesty, I also like the initial name better, but it does not reflect what the method does any more. 

In the first iteration the index was returned by / fed to methods, but now the methods read/write the global variable instead. I fixed the name in documentation, thanks for the notice!